src/main/java/uk/gov/hmcts/reform/slc/services/SendLetterClient.java

```diff
@@ -77,4 +80,22 @@ public void updateIsFailedStatus(UUID letterId) {
             );
         }
     }
+
+    /**
+     * Calls the Send Letter Producer service healthcheck.
+     *
+     * @return health status
+     */
+    public Health serviceHealthy() {
+        try {
+            ResponseEntity<InternalHealth> response = restTemplate
+                .getForEntity(sendLetterProducerUrl + "health", InternalHealth.class);
```

Comment:
It would be best to use some path builder, so that we wouldn't have to depend on sendLetterProducerUrl ending with `/`.

Reply:
it is dealth with in the constructor:
```java
this.sendLetterProducerUrl = appendIfMissing(sendLetterProducerUrl, "/");
```